Mathlib/Topology/Instances/CantorSet.lean

```diff
@@ -0,0 +1,93 @@
+/-
+Copyright (c) 2024 Jana Göken. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Artur Szafarczyk, Suraj Krishna M S, Jean-Baptiste Stiegler, Isabelle Dubois,
+Tomáš Jakl, Lorenzo Zanichelli, Alina Yan, Emilie Uthaiwat, Jana Göken,
+Filippo A. E. Nuccio
+-/
+import Mathlib.Topology.Metrizable.Basic
+import Mathlib.Topology.Algebra.GroupWithZero
+import Mathlib.Topology.Instances.Real
+
+/-!
+# Ternary Cantor Set
+
+This file defines the Cantor ternary set and proves a few properties.
+
+## Main Definitions
+
+* `preCantorSet n`: The order `n` pre-Cantor set, defined inductively as the union of the images
+  under the functions `(· ‌/ 3)` and `(2 * · / 3)`, with `preCantorSet 0 := Set.Icc 0 1`
+  * `cantorSet`: The ternary Cantor set, defined as the intersection of all pre-Cantor sets.
+-/
+
+/-- We define the preCantorSet as the images under the iterations of some functions. -/
```

Comment:
This still needs a better docstring. This comment was never addressed earlier from an earlier PR review.

Reply:
Should be addressed now. Please check the newest version.